src/__tests__/ParseObject-test.js

```diff
@@ -3543,3 +3543,23 @@ describe('ParseObject pin', () => {
     CoreManager.set('NODE_LOGGING', false);
   });
 });
+
+describe('ParseObject._getRequestOptions', () => {
```

Comment:
Well done, you have also added tests! We'd add the to a higher API like `Parse.Cloud.run` which invokes `_getRequestOptions`, rather than on `_getRequestOptions` itself. The test would be added to `/__tests__/Cloud-test.js`. You can also compact into a single "empty options" test:

```js
it('run passes with empty options', () => {
  const values = [
    null,
    undefined,
    {},
  ];    

  for (const value of values) {
    expect(Cloud.run('myfunction', {}, value)).not.toThrow();
    expect(CoreManager.getCloudController().run.mock.calls[0]).toEqual([
      'myfunction',
      {},
      {},
    ]);
  });
```

Note that I've deliberately left out the empty string value, see my other feedback.